### ISSUE TITLE
Adding manual seed for reproducibility

### ISSUE THREAD

**Issue by sarthakpati** [FETS-AI/Challenge#97]
Dear Organizers,

You may consider adding these lines to the internal codes. I think that it is also good for comparing team results at the end of the challenge. But I am not sure that they are enough :)

 torch.manual_seed(torch_manual_seed)
 torch.cuda.manual_seed_all(torch_manual_seed)
 torch.backends.cudnn.deterministic = True
 torch.backends.cudnn.benchmark = False

(maybe random_state can be fixed for train_test_split )

Best,
Ece

_Originally posted by @eceisik in https://github.com/FETS-AI/Challenge/discussions/80#discussioncomment-1004208_

**Comment by sarthakpati** [comment]
Tagging @psfoley and @alexey-gruzdev
